BenchmarkSample/benchmark/build.gradle

```diff
@@ -34,6 +36,7 @@ android {
     }
 }
 
+// [START benchmark_dependency]
```

Comment:
this seems like a large range, since it covers all of test dependencies - is this to be used in guide docs?